src/main/java/nextstep/subway/section/domain/Sections.java

```diff
@@ -0,0 +1,89 @@
+package nextstep.subway.section.domain;
+
+import nextstep.subway.exception.AlreadyExistDownStationException;
+import nextstep.subway.exception.DeleteSectionException;
+import nextstep.subway.exception.IsNotLastStationException;
+import nextstep.subway.exception.NotFoundStationException;
+import nextstep.subway.station.domain.Station;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Embeddable;
+import javax.persistence.OneToMany;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.LinkedList;
+import java.util.List;
+
+@Embeddable
+public class Sections {
+
+    @OneToMany(mappedBy = "line", cascade = CascadeType.ALL, orphanRemoval = true)
+    private List<Section> sections = new LinkedList<>();
+
+    public Sections() {
+    }
+
+    public List<Station> getStations() {
+        List<Station> stations = new ArrayList<>();
+
+        for (Section section : sections) {
+            stations.add(section.getUpStation());
+            stations.add(section.getDownStation());
+        }
+
+        return new ArrayList<>(new HashSet<>(stations));
+    }
+
+    public void addSection(Section section) {
+        if (size() == 0) {
```

Comment:
```suggestion
        if (sections.isEmpty()) {
```

로 하는 게 조금 더 직관적인 것 같아요 😅